external/fv3fit/fv3fit/reservoir/model.py

```diff
@@ -1,19 +1,32 @@
 import fsspec
 import numpy as np
 import os
-from typing import Optional, Iterable, Hashable
+from typing import Optional, Iterable, Hashable, Union
 import yaml
 
+import fv3fit
 from fv3fit import Predictor
 from .readout import ReservoirComputingReadout
 from .reservoir import Reservoir
 from .domain import RankDivider
 from fv3fit._shared import io
 from .utils import square_even_terms
-from .autoencoder import Autoencoder
+from .transformers import Autoencoder, SkTransformer, ReloadableTransfomer
 from ._reshaping import flatten_2d_keeping_columns_contiguous
 
 
+def _load_transformer(path) -> Union[Autoencoder, SkTransformer]:
```

Comment:
return type of `ReloadableTransformer`?

Reply:
I did try that first, but mypy was unhappy.

Reply:
The altar of mypy is absolute 😂